test/integration/model/bulk-create.test.js

```diff
@@ -745,6 +745,104 @@ describe(Support.getTestDialectTeaser('Model'), () => {
             this.User.bulkCreate(data, { updateOnDuplicate: [] }),
           ).to.be.rejectedWith('updateOnDuplicate option only supports non-empty array.');
         });
+
+        if (current.dialect.supports.inserts.conflictFields) {
+          it('should respect the conflictFields option', async function () {
+            const Model = this.sequelize.define(
```

Comment:
Because I've seen tests modify the base Model class, renaming this class will help me quickly determine if there is a red flag in the test in the future :)

```suggestion
            const Permission = this.sequelize.define(
```